controlplane/test/namespace.test.ts

```diff
@@ -187,27 +192,63 @@ describe('Namespaces', (ctx) => {
     await createFederatedGraph(client, fedGraphName, prod, [joinLabel(label)], 'http://localhost:8080');
     await createFederatedGraph(client, fedGraphName, dev, [joinLabel(label)], 'http://localhost:8081');
 
-    const graphsRes = await client.getFederatedGraphs({});
+    /**
+     * Verify that all graphs are created
+     */
+
+    let graphsRes = await client.getFederatedGraphs({
+      namespace: prod,
+    });
     expect(graphsRes?.response?.code).toBe(EnumStatusCode.OK);
-    expect(graphsRes?.graphs.length).toBe(2);
```

Comment:
These tests were there to ensure the behaviour of list to be unchanged. I admit it is my fault for not adding a comment in the rpc.